devtools/x/src/context.rs

```diff
@@ -20,6 +21,11 @@ impl XContext {
     pub fn with_config(config: Config) -> Self {
         Self {
             core: XCoreContext::new(project_root()),
+            //not happy with the cloning action here....
```

Comment:
Could you mark this as XXX? The way to separate it out would probably be to have data structures with/without.

Reply:
I'm not sure what you mean by "Could you mark this as XXX?"

I'll simply remove the line.